dataprofiler/tests/labelers/test_data_processing.py

```diff
@@ -671,6 +673,22 @@ def test_process(self):
             self.assertTrue((expected[0] == output[0]).all())
             self.assertTrue((expected[1] == output[1]).all())
 
+    def test_process_input_checks(self):
+        prep = CharPreprocessor()
+        multi_dim_msg = re.escape("Multidimensional data given to "
```

Comment:
I dont think you need re.escape(), I think you should remove it and remove the re import

Reply:
I use it because of my use of parenthesis and characters like !, =